src/Announcements-Core/SubscriptionRegistry.class.st

```diff
@@ -52,6 +52,14 @@ SubscriptionRegistry >> deliver: anAnnouncement to: subs startingAt: startIndex
 				self deliver: anAnnouncement to: subs startingAt: index + 1 ]]
 ]
 
+{ #category : #testing }
+SubscriptionRegistry >> handleSubscriberClass: eventClass [
+	"Return true if the receiver has a callback subscripbed for the event class"
+	^ subscriptions anySatisfy: [ :sub | 
+		(sub subscriber class == eventClass) or: 
+			[ sub subscriber class includesBehavior: eventClass ] ]
```

Comment:
It should be at least #handles... with ending s as a common convention for such methods.
But generally name does not sound good in English. It should be something like
```Smalltalk
announcer isSubscribedToAnnouncementClass:  eventClass
``` 
and 
```Smalltalk
subscriptionRegistry includesSubscriptionsToAnnouncementClass:  eventClass
```